hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/SCMPipelineManager.java

```diff
@@ -158,6 +158,8 @@ public synchronized Pipeline createPipeline(
       throw idEx;
     } catch (IOException ex) {
       metrics.incNumPipelineCreationFailed();
+      LOG.error("Pipeline creation failed due to exception: "
```

Comment:
This can be changed to: LOG.error("Pipeline creation failed", ex);

Reply:
Updated.